docs/advanced.rst

```diff
@@ -11,12 +11,12 @@ Command line interface
 .. code-block:: console
 
     usage: grocker [-h] [-c CONFIG] [-r RUNTIME] [-e ENTRYPOINT_NAME]
-                   [--volume VOLUMES] [--port PORTS] [--pip-conf <file>]
-                   [--pip-constraint <file>] [--docker-image-prefix <url>]
-                   [--image-base-name <name>] [-n <name>]
-                   [--result-file <filename>]
-                   [--purge {all,builders,dangling,runners}] [--version] [-v]
-                   <action> [<action> ...] <release>
+               [--volume VOLUMES] [--port PORTS] [--pip-conf <file>]
+               [--pip-constraint <file>] [--docker-image-prefix <url>]
+               [--image-base-name <name>] [-n <name>]
+               [--result-file <filename>]
+               [--purge {old,old:runner,all,all:runner}] [--version] [-v]
+               <action> [<action> ...] <release>
```

Comment:
Wondering if we should be using something like https://github.com/pypa/pip/blob/master/docs/pipext.py#L14-L22
Which automatically creates: https://pip.pypa.io/en/stable/reference/pip_install/#usage (and keep it in sync).